manifests/s/Smartedu/Smartedu/1.0.0/Smartedu.Smartedu.locale.zh-CN.yaml

```
@@ -0,0 +1,19 @@
+# Created using wingetcreate 1.2.6.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.4.0.schema.json
+
+PackageIdentifier: Smartedu.Smartedu
+PackageVersion: 1.0.0
+PackageLocale: zh-CN
+Publisher: 中华人民共和国教育部
```

```suggestion
Publisher: 智慧中小学
```
This should match what is written into control panel